### PR TITLE
Kiosk tutorials update

### PR DESCRIPTION
**Pull request by AlanGriffiths**
## Done

Update examples and utilities used in mir-kiosk based tutorials

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8016/](http://0.0.0.0:8016/)

### CODE REVIEW

File: tutorials/iot/ubuntu-kiosk/wayland-kiosk.md [hunk 1]
```diff
@@ -190,15 +190,23 @@ grade: devel
 apps:
   glmark2-example:
     command: "usr/bin/glmark2-wayland --fullscreen"
-    plugs:
-      - opengl
-      - wayland
+
+plugs:
+  opengl:
+  wayland:
```
Comment (Saviq) [inline tutorials/iot/ubuntu-kiosk/wayland-kiosk.md]: Why this change?

File: tutorials/iot/ubuntu-kiosk/wayland-kiosk.md [hunk 2]
```diff
@@ -533,39 +551,40 @@ We will revisit this later to ensure the snap can be compiled to function on oth
...
+```yaml
+  mir-kiosk-snap-launch:
+    plugin: dump
+    source: https://github.com/MirServer/mir-kiosk-snap-launch.git
+    override-build:  $SNAPCRAFT_PART_BUILD/build-with-plugs.sh opengl wayland
 ```
 
 
-…and point `command:` in the snapcraft.yaml file to it (don't forget to install the script in the snap!).
+…and change `command:` in the snapcraft.yaml file to it (don't forget to install the script in the snap!).
```
Comment (Saviq) [inline tutorials/iot/ubuntu-kiosk/wayland-kiosk.md]: Since it's `dump`ed, does "don't forget…" apply still?

File: tutorials/iot/ubuntu-kiosk/wayland-kiosk.md (hunk 1 above)
Comment (AlanGriffiths) [reply tutorials/iot/ubuntu-kiosk/wayland-kiosk.md]: It simplifies the final .yaml

File: tutorials/iot/ubuntu-kiosk/wayland-kiosk.md (hunk 2 above)
Comment (AlanGriffiths) [reply tutorials/iot/ubuntu-kiosk/wayland-kiosk.md]: Fixed
